source/codegen/metadata/nidcpower/CHANGES.md

```diff
@@ -54,6 +52,29 @@ Following enums were defined :
     - NIDCPOWER_VAL_POWER_ALLOCATION_MODE_AUTOMATIC to NIDCPOWER_VAL_AUTOMATIC
     - NIDCPOWER_VAL_POWER_ALLOCATION_MODE_MANUAL to NIDCPOWER_VAL_MANUAL
 
+The following enums were removed :
+- InternalReference
+- CalibrationCloseAction
+
+## functions.py
+
+Following calibration functions were removed from the metadata :
+- ChangeExtCalPassword
+- GetCalUserDefinedInfoMaxSize
+- SetCalUserDefinedInfo
+- GetCalUserDefinedInfo
+- InitExtCal
```

Comment:
Remove other doc in this file related to InitExtCal and CloseExtCal before submitting

Reply:
Done